static/app/views/sharedGroupDetails/sharedGroupHeader.tsx

```diff
@@ -33,6 +41,21 @@ function SharedGroupHeader({group}: Props) {
               />
             )}
           </ShortIdWrapper>
+          {event && (event.dateCreated ?? event.dateReceived) && (
+            <TimeStamp data-test-id="sgh-timestamp">
+              {t('Last seen ')}
+              <EventTimeLabel>
+                <Tooltip
+                  isHoverable
+                  showUnderline
+                  title={<EventCreatedTooltip event={event} />}
+                  overlayStyle={{maxWidth: 300}}
+                >
+                  <DateTime date={date.toLocaleString()} />
```

Comment:
don't pass locale strings around should be able to pass the date object or toIsoString
```suggestion
                  <DateTime date={date} />
```